Source/WebKit/UIProcess/WebAuthentication/AuthenticatorManager.cpp

```diff
@@ -508,7 +508,7 @@ void AuthenticatorManager::runPresenter()
 void AuthenticatorManager::runPresenterInternal(const TransportSet& transports)
 {
     auto& options = m_pendingRequestData.options;
-    m_presenter = makeUnique<AuthenticatorPresenterCoordinator>(*this, getRpId(options), transports, getClientDataType(options), getUserName(options));
+    m_presenter = makeUniqueWithoutRefCountedCheck<AuthenticatorPresenterCoordinator>(*this, getRpId(options), transports, getClientDataType(options), getUserName(options));
```

Comment:
This cannot be correct. m_presenter gets nulled out in `AuthenticatorManager::clearState()`, therefore `AuthenticatorPresenterCoordinator` cannot forward its ref-counting to `AuthenticatorManager`.

Reply:
Thanks. Yeah, I totally misunderstood the lifecycle. Changed to RefCountedAndCanMakeWeakPtr.